serialization.md

```diff
@@ -74,7 +74,7 @@ This specification uses the following terms:
         A collection of tags grouped under a common prefix (the name component before <code>:</code>).
 	For example, in an image tagged with the name <code>my-app:3.1.4</code>, <code>my-app</code> is the <i>Repository</i> component of the name.
 	A repository name is made up of slash-separated name components, optionally prefixed by a DNS hostname.
-	The hostname must follow comply with standard DNS rules, but may not contain <code>_</code> characters.
+	The hostname must comply with standard DNS rules that apply to IDNA labels as defined in <a href="https://tools.ietf.org/html/rfc5890#section-2.3">RFC 5890</a>.
```

Comment:
I [still][1] think we want to call out A-label, U-label, or NR-LDH label to avoid forbidding all-ASCII, non-A label, repository names.

[1]: https://github.com/opencontainers/image-spec/pull/163#discussion_r76098406